From: Thomas Perret <thomas.perret@phyx.fr>
Date: Fri, 9 Sep 2022 00:33:40 +0200
Subject: Move from nose to pytest
Forwarded: no

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index b0f2750..e010f86 100755
--- a/setup.py
+++ b/setup.py
@@ -90,5 +90,5 @@ setup(
     install_requires=[
         "Pillow",
     ],
-    setup_requires=['nose>=1.0'],
+    setup_requires=['pytest'],
 )
